### DISCUSSION TITLE
Updating pocketbase when user changes their bio

### DISCUSSION THREAD

**Discussion by sarbesh0x01** [pocketbase/pocketbase#4733] [Q&A]
I have this function for user to change their bio but it gives Uer.id as undefined

**Comment by ganigeorgiev** [comment]
Please don't post code as screenshot.

Based on your description your error doesn't seem PocketBase related.

Make sure that the `User` object is defined and loaded.

**Comment by sarbesh0x01** [comment]
Ok
